conans/client/cmd/uploader.py

```diff
@@ -181,10 +187,15 @@ def _collect_packages_to_upload(self, refs, confirm, remotes, all_packages, quer
                                              "no packages can be uploaded" % str(ref))
                 prefs = []
                 # Gather all the complete PREFS with PREV
-                for package_id in packages_ids:
+                for package in packages_ids:
+                    package_id = package.split("#")[0] if "#" in package else package
```

Comment:
pkg_id, prev= package.split("#") if "#" in package else (package, None)

 Call it prev , it is the convention like rrev